tests/test_properties.py

```diff
@@ -135,3 +185,47 @@ async def test_roundtrip_array_metadata(
 #     nparray = data.draw(np_arrays)
 #     zarray = data.draw(arrays(arrays=st.just(nparray)))
 #     assert_array_equal(nparray, zarray[:])
+
+
+@given(data=st.data(), zarr_format=zarr_formats)
+def test_meta_roundtrip(data: st.DataObject, zarr_format: int) -> None:
```

Comment:
can we update the `test_roundtrip_array_metadata` test further up in the file instead please?

Reply:
Honestly, missed it. Will do.

Reply:
OK, updated test names and added docstrings to clarify what's going on